sympy/solvers/inequalities.py

```diff
@@ -985,6 +985,18 @@ def reduce_inequalities(inequalities, symbols=[]):
 
     # solve system
     rv = _reduce_inequalities(inequalities, symbols)
+    as_dict = flags.get('dict', False)
```

Comment:
Add this as an explicit keyword argument, rather than `**flags`.

Reply:
Well I used `**flags` because maybe there'd be more flags in future which may need to be checked for. So should I change it to `dict` being passed.

Reply:
Can you change it to an explicit argument? The problem with using `**flags` is that it allows other keyword arguments to be passed without raising an error (I realise that other functions in this module do that but we shouldn't make it worse).

Also can you describe the dict parameter in the doc-string and perhaps add a doctest showing what it does?